models/models.js

```diff
@@ -21,3 +21,26 @@ exports.selectArticleByID = (article_id) => {
         return article.rows;
     })
 }
+
+exports.selectArticles = () => {
+    return db
+    .query(`
+    SELECT articles.author, title, articles.article_id, topic, articles.created_at, articles.votes, article_img_url, COUNT(comments.article_id) AS comment_count
+    FROM articles
+    LEFT JOIN comments
+    ON articles.article_id = comments.article_id
+    GROUP BY articles.article_id
+    ORDER BY created_at DESC;`)
+    .then(({rows}) => {
+        console.log(rows);
```

Comment:
Remove all console.logs before you push code

Reply:
done